src/main/scala/com/codacy/analysis/cli/command/CLIApp.scala

```diff
@@ -114,6 +114,10 @@ final case class Analyse(
   upload: Int @@ Counter = Tag.of(0),
   @ExtraName("p") @ValueDescription("The number of tools to run in parallel")
   parallel: Option[Int] = Option.empty,
+  @ExtraName("n") @ValueDescription(
+    s"""If the access to the network is allowed, the cli will run tools that need it. (${Tool.allEnterpriseToolShortNames
```

Comment:
Maybe start a new sentence before dumping all the tools like "Supported tools that need internet: ".